### PR TITLE
Test

### PR DESCRIPTION
**Pull request by josefsys**
bylo to bezva

### CODE REVIEW

**Comment by josefsys** [comment]
comment

File: firstFile/secFile [hunk 1]
```diff
@@ -0,0 +1 @@
+efisfbu
```
Comment (Hyperkid123) [inline firstFile/secFile]: This does not look like hash of commit. Also it was supposed to be in the same file as your joke (btw the joke is good)

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]
You are also missing third commit with come git diff.

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]
You have added all the changes that were made on master branch as your. You can fix it by rebasing and removing commits that are not yours.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request closed without merging**